### PR TITLE
Exclude tests using jdk.test.whitebox.WhiteBox class

### PR DESCRIPTION
**Pull request by fengxue-IS**
Closes: https://github.com/eclipse-openj9/openj9/issues/18049

### CODE REVIEW

File: openjdk/excludes/ProblemList_openjdk21-openj9.txt [hunk 1]
```diff
@@ -549,6 +550,7 @@ serviceability/jvmti/thread/GetStackTrace/getstacktr05/getstacktr05.java https:/
 serviceability/jvmti/vthread/VThreadNotifyFramePopTest/VThreadNotifyFramePopTest.java https://github.com/adoptium/aqa-tests/issues/1297 aix-all
 serviceability/jvmti/vthread/FollowReferences/VThreadStackRefTest.java#no-vmcontinuations https://github.com/eclipse-openj9/openj9/issues/17712 generic-all
 serviceability/jvmti/RedefineClasses/RedefinePreviousVersions.java https://github.com/adoptium/aqa-tests/issues/1297 generic-all
+serviceability/jvmti/vthread/ToggleNotifyJvmtiTest/ToggleNotifyJvmtiTest.java https://github.com/eclipse-openj9/openj9/issues/18049 generic-all
```
Comment (babsingh) [inline openjdk/excludes/ProblemList_openjdk21-openj9.txt:553]: Permanently excluded tests are disabled using the following issue: https://github.com/adoptium/aqa-tests/issues/1297. The OpenJ9 issue can be referred in the commit message.
```suggestion
serviceability/jvmti/vthread/ToggleNotifyJvmtiTest/ToggleNotifyJvmtiTest.java https://github.com/adoptium/aqa-tests/issues/1297 generic-all
```

**Review by babsingh — COMMENTED** [review]
We also need to temporarily exclude other tests listed in https://github.com/eclipse-openj9/openj9/issues/18037#issuecomment-1699714194 using their corresponding OpenJ9 issues.

File: openjdk/excludes/ProblemList_openjdk21-openj9.txt [hunk 2]
```diff
@@ -549,6 +550,7 @@ serviceability/jvmti/thread/GetStackTrace/getstacktr05/getstacktr05.java https:/
 serviceability/jvmti/vthread/VThreadNotifyFramePopTest/VThreadNotifyFramePopTest.java https://github.com/adoptium/aqa-tests/issues/1297 aix-all
 serviceability/jvmti/vthread/FollowReferences/VThreadStackRefTest.java#no-vmcontinuations https://github.com/eclipse-openj9/openj9/issues/17712 generic-all
```
Comment (babsingh) [inline openjdk/excludes/ProblemList_openjdk21-openj9.txt:551]: `VThreadStackRefTest` already has an exclude for the `no-vmcontinuations` variant. You can remove `#no-vmcontinuations` to also disable the `default` variant of the test case.

Comment (babsingh) [reply openjdk/excludes/ProblemList_openjdk21-openj9.txt:551]:
```suggestion
serviceability/jvmti/vthread/FollowReferences/VThreadStackRefTest.java https://github.com/eclipse-openj9/openj9/issues/17712 generic-all
```

**Comment by babsingh** [comment]
> We also need to temporarily exclude the other tests listed in https://github.com/eclipse-openj9/openj9/issues/18037#issuecomment-1699714194 using their corresponding OpenJ9 issues.

I see another PR, https://github.com/adoptium/aqa-tests/pull/4738, which addresses the above feedback.

File: openjdk/excludes/ProblemList_openjdk21-openj9.txt (hunk 2 above)
Comment (babsingh) [reply openjdk/excludes/ProblemList_openjdk21-openj9.txt:551]: This comment is relevant for https://github.com/adoptium/aqa-tests/pull/4738.

File: openjdk/excludes/ProblemList_openjdk21-openj9.txt (hunk 1 above)
Comment (fengxue-IS) [reply openjdk/excludes/ProblemList_openjdk21-openj9.txt]: updated

**Review by babsingh — APPROVED** [review]
@llxia @LongyuZhang Requesting your review.

**Review by LongyuZhang — APPROVED** [review]
LGTM

**Review by llxia — APPROVED** [review]
LGTM

**Pull request merged**